### PR TITLE
Added bot emoticons command

### PR DESCRIPTION
**Pull request by brasstax**
Basic emoticons added for various feelings. Includes shrug, table-flipping, and unflipping tables.

### CODE REVIEW

File: sopel/modules/emoticons.py [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+    bot.say('¯\_(ツ)_/¯')
+
+
+@commands('happy')
+@example('.happy', 'ᕕ( ᐛ )ᕗ')
+def happy(bot, trigger):
+    bot.say('ᕕ( ᐛ )ᕗ')
+
+
+@commands('tableflip')
```
[the remark this answers is not in the document]
Comment (brasstax) [reply sopel/modules/emoticons.py]: Amended regardless because I can. `.shrug`

**Comment by brasstax** [comment]
>Needs to be rebased. Commit cf092bd sneaked into this feature branch from #1317. Other than that, looks pretty straightforward. I'm 👍 on this once the extraneous commit is removed.
Done. (And that's a good lesson on why I shouldn't work on my own `master` branch in favor of an actual branch.

**Comment by dgw** [comment]
Hey @brasstax, I just realized this PR doesn't include the usual module header. The core modules vary a bit on how it looks (especially the older ones), but most commonly it's something like:

https://github.com/sopel-irc/sopel/blob/ade0aeec8368e72e29cc0f0197f6bf0638f0dd9e/sopel/modules/tld.py#L2-L8

Do you want to add it yourself and amend the PR, or should I just add it before merging? I'd put:

````python
"""
emoticons.py - Sopel Emoticons Module
Copyright 2018, brasstax
Licensed under the Eiffel Forum License 2

https://sopel.chat
"""
````

as I see no reason to require email address or website unless you want to put that info in.

**Comment by brasstax** [comment]
Could you add it? I'm on vacation this week. I'll remember next time.

Thanks!

**Comment by dgw** [comment]
Sure thing, next time I've got the repo open (on a computer, that is). :)

**Comment by dgw** [comment]
Should have skipped CI on that, oh well.

**Pull request merged**